extensions/ql-vscode/src/extension.ts

```diff
@@ -467,9 +468,14 @@ async function activateWithInstalledDistribution(
     selectedQuery: Uri | undefined,
     progress: ProgressCallback,
     token: CancellationToken,
+    databaseQuickPick: DatabaseItem | undefined,
```

Comment:
I suggest naming this variable to indicate that it is a database item, not the fact that it comes from a quick pick. The fact that it comes from a quick pick is something that matters only to the calling function. From the perspective of this function: "if you give me a database item I'll use it, if not I'll default to the currently selected database".
```suggestion
    databaseItem: DatabaseItem | undefined,
```